web/src/emoji_picker.ts

```diff
@@ -216,9 +236,9 @@ function get_rendered_emoji(section, index) {
     return undefined;
 }
 
-export function is_emoji_present_in_text(text, emoji_dict) {
+export function is_emoji_present_in_text(text: string, emoji_dict: Emoji): boolean {
     // fetching emoji details to ensure emoji_code and reaction_type are present
-    const emoji_info = emoji.get_emoji_details_by_name(emoji_dict.name);
+    const emoji_info = emoji.get_emoji_details_by_name(emoji_dict.name ?? "");
```

Comment:
Why `??` coalescing `emoji_dict.name` should always be present given that you have typed is as always `string`